Cargo.toml

```diff
@@ -43,4 +43,5 @@ native-tls = "0.2.11"
 postgres-native-tls = "0.5.0"
 prometheus = "0.13.3"
 lazy_static = "1.4.0"
-dotenv = "0.15.0"
\ No newline at end of file
+flume = "0.10.14"
```

Comment:
why introduce another dependency to our stack? why not use crossbeam / async channel?

Reply:
this is a mpmc channel. We need a simple spmc solution to distribute tasks across multiple threads.
Crossbeam doesn't work great with tokio as discussed with galactus. Therefore `flume`

Reply:
Can replace this with async_channel